preview/app/controllers/components_controller.rb

```diff
@@ -5,11 +5,11 @@ def show
 
     if template_exists?(template)
       render template,
-             locals: {
-               framework: framework,
-               component_parameters:
-                 component_parameters(component_name.underscore)
-             }
+              locals: {
+                framework: framework,
+                component_parameters:
+                  component_parameters(component_name.underscore)
+              }
```

Comment:
was this intentional?

Reply:
yeah my linter was mad, so I fixed it.

Reply:
What linter do you use for these files? I actually don't know what's the right indent here.

Reply:
it was using an odd number (3 instead of 2) of indented spaces so the fix is correct.
I use Solargraph on VSCode which does the linting.